.gitlab-ci.yml

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2016-19, Lawrence Livermore National Security, LLC and Serac
+# project contributors. See the COPYRIGHT file for details.
+##############################################################################
```

Comment:
switch to (2014-2020) and Conduit project contributors

Reply:
Does it mean all the sources in conduit should switch to 2020, or is it done incrementally when the given file is modified?
Serac to Conduit: OK